src/open_inwoner/templates/pages/category/detail.html

```diff
@@ -3,7 +3,9 @@
 
 {% block header_image %}
 {% if object.image %}
-    <img class="main-image main-image--start-after" src="{{ object.image.url }}" alt="Header image">
+    <aside aria-label="{% trans "Header afbeelding" %}">
+        <img class="main-image main-image--start-after" src="{{ object.image.url }}" alt="Header image">
```

Comment:
Can we use the imaga's alt text here?

Reply:
Attempted to do this but might have to be applied on other places too